bin/env.sh

```diff
@@ -2,16 +2,21 @@
 
 export volume=edgexfoundry/docker-edgex-volume
 export consul=consul:1.1.0
-export coreMetadata=nexus3.edgexfoundry.org:10004/docker-core-metadata-go
-export coreConfigseed=nexus3.edgexfoundry.org:10004/docker-core-config-seed-go:master
-export mongo=edgexfoundry/docker-edgex-mongo
+export coreConfigseed=edgexfoundry/docker-core-config-seed-go:0.6.0
+
+export mongo=edgexfoundry/docker-edgex-mongo:0.6.0
 export mongoSeed=edgexfoundry/docker-edgex-mongo-seed
-export supportLogging=nexus3.edgexfoundry.org:10004/docker-support-logging
-export coreData=nexus3.edgexfoundry.org:10004/docker-core-data-go
-export coreCommand=nexus3.edgexfoundry.org:10004/docker-core-command-go
-export supportScheduler=edgexfoundry/docker-support-scheduler
-export exportClient=edgexfoundry/docker-export-client
-export exportDistro=edgexfoundry/docker-export-distro
-export supportRulesengine=edgexfoundry/docker-support-rulesengine
-export deviceVirtual=edgexfoundry/docker-device-virtual
 
+export coreData=edgexfoundry/docker-core-data-go:0.6.0
+export coreMetadata=edgexfoundry/docker-core-metadata-go:0.6.0
+export coreCommand=edgexfoundry/docker-core-command-go:0.6.0
+
+export supportLogging=edgexfoundry/docker-support-logging-go:0.6.0
+export supportNotification=edgexfoundry/docker-support-notifications:0.6.0
+export supportScheduler=edgexfoundry/docker-support-scheduler:0.6.0
+export supportRulesengine=edgexfoundry/docker-support-rulesengine:0.6.0
+
+export deviceVirtual=edgexfoundry/docker-device-virtual:0.6.0
+
+export exportClient=edgexfoundry/docker-export-client:0.2.1
```

Comment:
Is there a specific reason this is still 0.2.1? We have a 0.6.0 Go implementation now.
https://hub.docker.com/r/edgexfoundry/docker-export-client-go/tags/

Reply:
Because this issue https://github.com/edgexfoundry/support-rulesengine/issues/26 , support-rulesengine can't startup
I notice PR https://github.com/edgexfoundry/support-rulesengine/pull/27 merged ,
I had changed to export-client-go , it will work after docker images update